app/Http/Controllers/DashboardController.php

```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
 
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
+    {
+        $loadouts = collect();
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
```

Comment:
We can definitely extract this and we can just pass a parameter for order and last n weeks or days. Can add a TODO if you aren't feeling it and I can knock it out.